tests/settings/e2e/steps/currency-display-settings.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import { When, Then} from 'cucumber';
+
+const CONVERTED_CURRENCY_WALLET_DISPLAY_GENERAL = '//*[@class="WalletSummaryCurrency_currencyRate"]';
+const DISPLAY_BALANCE_OTHER_CURRENCYS_TOGGLE_STATUS_OFF = '//*[@class="SimpleSwitch_switch SwitchOverrides_switch"]';
+const DISPLAY_BALANCE_OTHER_CURRENCYS_TOGGLE_STATUS_ON = '//*[@class="NormalSwitch_component SimpleSwitch_root SwitchOverrides_root SimpleSwitch_checked SwitchOverrides_checked"]';
+const SELECT_CURRENCY_DROPDOWN = '//*[@label="Select currency"]';
+
+When(/^I open currency selection dropdown$/, function() {
+  return this.waitAndClick(SELECT_CURRENCY_DROPDOWN);
```

Comment:
I think that you can use just `click` since you don't need to wait for something. Please try... :)

Reply:
I must have not implemented this right ? 
<img width="1195" alt="Screenshot 2021-07-19 at 16 54 06" src="https://user-images.githubusercontent.com/56027085/126189706-71ef0396-50ea-43ed-8d21-250805609a74.png">
<img width="505" alt="Screenshot 2021-07-19 at 16 54 16" src="https://user-images.githubusercontent.com/56027085/126189710-d0d8e470-cc1d-4a76-8204-703251afc1af.png">

Reply:
@ManusMcCole for native methods, it needs `this.client. ...`. `waitAndClick` is a helper method created by us, therefore `client` is not needed.